mmv1/products/discoveryengine/product.yaml

```diff
@@ -1,22 +0,0 @@
-# Copyright 2023 Google Inc.
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
---- !ruby/object:Api::Product
-name: DiscoveryEngine
-display_name: Discovery Engine
-versions:
-  - !ruby/object:Api::Product::Version
-    name: ga
-    base_url: https://discoveryengine.googleapis.com/v1alpha/
```

Comment:
We're rolling out v1 next week, which should be ready by next Friday. We're using API version selector to build this and there should be no breaking changes once we update to v1 later. On the backend both versions are sharing the same implementations. Could we just wait until next week and we update this to v1 instead of rolling it back?

Reply:
We caught up over chat and agreed to move forward with the revert. See https://github.com/GoogleCloudPlatform/magic-modules/pull/9682#issuecomment-1899331791.